app/backend/templates/v2/activeness_probe.mjml

```diff
@@ -0,0 +1,19 @@
+<mj-section padding="0px">
+  <mj-column>
+    <mj-text>Hi {{ user.name|v2esc }},</mj-text>
+    <mj-text>It's been a while since you logged in to Couchers.org, and we just wanted to check if you are still actively hosting surfers?</mj-text>
```

Comment:
```suggestion
    <mj-text>We noticed you haven’t logged in for a while and wanted to check if you're still open to hosting surfers.</mj-text>
```

Reply:
Not 100% sure it's 15 days the limit so please check me there.

Reply:
Or use `ACTIVENESS_PROBE_EXPIRY_TIME` probably better than hardcoding the days in case we change it.